taskcluster/ci/beetmover/kind.yml

```diff
@@ -48,3 +49,50 @@ jobs:
             kind: build
             tier: 1
             platform: windows/x86_64
+    addons-bundle:
+        worker-type: beetmover
+        worker:
+            chain-of-trust: true
+            max-run-time: 3600
+        run-on-tasks-for: [action:release-promotion]
+        release-artifacts:
+            - guide_01_how_to_vpn.rcc
+            - guide_02_is_my_vpn_working_correctly.rcc
+            - guide_03_adding_and_removing_devices.rcc
+            - guide_04_connecting_external_devices.rcc
+            - message_survey_28d_desktop.rcc
+            - message_survey_28d_mobile.rcc
+            - message_survey_staging_10s.rcc
+            - message_survey_staging_4s.rcc
+            - message_survey_staging_locale.rcc
+            - tutorial_01_get_started.rcc
+            - tutorial_02_connect_on_startup.rcc
+        dependencies:
+            build: build-addons-bundle
+        if-dependencies: [build]
+        attributes:
+            build-type: addons/opt
+        treeherder:
+            symbol: BM(addons-bundle)
+            kind: build
+            tier: 1
+            platform: addons/opt
+    addons-manifest:
+        worker-type: beetmover
+        worker:
+            chain-of-trust: true
+            max-run-time: 3600
```

Comment:
An hour seems a bit long, I might change it to 30min. But I guess that's what the other tasks have already, so we could do it in a follow-up.

Reply:
Done